image_client/picturae_importer.py

```diff
@@ -24,9 +24,32 @@ class PicturaeImporter(Importer):
            along with attached images
     """
 
-    def __init__(self, paths, date_string=None, istesting=False):
+    def __init__(self, paths, date_string=None, istest=False):
         super().__init__(picturae_config, "Botany")
 
+        self.setting_init_variables(date_string=date_string, paths=paths)
+
+        # running csv create
+        CsvCreatePicturae(date_string=self.date_use)
+
+        self.file_path = f"PIC_upload/PIC_record_{self.date_use}.csv"
+
+        self.record_full = pd.read_csv(self.file_path)
+
+        self.batch_size = len(self.record_full)
+
+        self.batch_md5 = generate_token(starting_time_stamp, self.file_path)
+
+        if istest is False:
+            self.run_all_methods()
+
+
+    def setting_init_variables(self, date_string, paths):
```

Comment:
same dilemma as init_all_vars function above , not sure if will be useful in unittests or not.